test/solveEquation/solveEquation.test.js

```diff
@@ -88,6 +89,12 @@ describe('solveEquation for =', function () {
     ['x^2 (x - 5)^2 = 0', 'x = [0, 0, 5, 5]'],
     ['x^2 = 0', 'x = [0, 0]'],
     ['(x+2)^2 -x^2 = 4(x+1)', '4 = 4'],
+    ['2/x = 1', 'x = 2'],
+    ['2/(4x) = 1', 'x = 1/2'],
+    ['2/(8 - 4x) = 1/2', 'x = 1'],
+    ['2/(1 + 1 + 4x) = 1/3', 'x = 1'],
+    ['(3 + x) / (x^2 + 3) = 1', 'x = [0, 1]'],
+    ['6/x + 8/2x = 10', NO_STEPS],
```

Comment:
I realized that the test I wanted from this was actually missing parens in `8/(2x)`.  I've added that and adjusted the test result.

I didn't leave the previous test case without parens because it started to look similar to the test cases in the other TODO related to complex numbers.  I can add it back with a comment if you think it's a separate scenario